src/plugins/discover_legacy/public/application/angular/discover.js

```diff
@@ -480,7 +481,23 @@ function discoverController($element, $route, $scope, $timeout, $window, Promise
       },
     };
 
+    const newDiscover = {
+      id: 'discover-new',
+      label: i18n.translate('discover.localMenu.newDiscoverTitle', {
+        defaultMessage: 'New Discover',
```

Comment:
The issue for this change is still unresolved. This will most likely become a toggle. I added this for now simply to make development easier to validate changes between the two discovers.